extensions/tailscale/src/devices.tsx

```diff
@@ -12,7 +12,7 @@ function DeviceList() {
         const _list = getDevices(status);
         setDevices(_list);
       } catch (error) {
-        setError(getErrorDetails(error, "Couldn’t load device list."));
+        setError(getErrorDetails(error, "Couldn't load device list."));
```

Comment:
Mind reverting this line, since the old version was the correct punctuation? (An apostrophe rather than a hash mark)